webapp/brilla-frontend/.gitignore

```diff
@@ -33,4 +34,3 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
-package-lock.json
```

Comment:
should we not include this here? else folks could accidentally push package-lock files and mess up the environment for others.

Reply:
We don't need the  package-lock.json to be in the .gitignore. Having it  committed ensures that everyone can have the correct dependencies and versions installed during the build process.